src/components/Send/SendScreenNavigator.js

```diff
@@ -30,9 +32,22 @@ const SendScreenNavigator = createStackNavigator(
             style={styles.qrButton}
             onPress={() =>
               navigation.navigate(SEND_ROUTES.ADDRESS_READER_QR, {
-                onSuccess: (address) => {
-                  const handler = navigation.getParam('onScanAddress')
-                  handler && handler(address)
+                onSuccess: (stringQR) => {
+                  const regex = /(cardano):([a-zA-Z1-9]\w+)\?/
```

Comment:
I would keep it `a-zA-Z1-9` and do validation of address-specific rules after this regex passes because v2 validation rules are different than Shelley validation rules so it would be weird to duplicate this logic inside the regex.